ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master__nightly.yaml

```diff
@@ -0,0 +1,63 @@
+canonical_go_repository: github.com/openshift/tektoncd-pipeline-operator
+
+base_images:
+  os:
+    name: centos
+    namespace: openshift
+    tag: '7'
+  operator-registry:
+    name: "4.1"
+    namespace: ocp
+    tag: operator-registry
+
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      memory: 250Mi
+      cpu: 100m
+
+images:
+- from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/tektoncd-pipeline-operator/out/operator 
+  dockerfile_path: openshift-ci/Dockerfile.deploy
+  to: tektoncd-pipeline-operator
+- from: operator-registry
+  dockerfile_path: openshift-ci/Dockerfile.registry.intermediate
+  to: operator-registry-base
+- from: operator-registry-base
+  dockerfile_path: openshift-ci/Dockerfile.registry.build
+  to: tektoncd-pipeline-operator-registry
+- from: src
+  dockerfile_path: openshift-ci/Dockerfile.upgrade.build
+  to: upgrade-manifest-build
+
+promotion:
```

Comment:
we need this promotion @droslean. maybe a different image and repo name. Because we want to mirror these images quay afterwards

Reply:
you want to promote the images on every periodic run?

Reply:
yes @droslean